tests/unit/Rules/PerfectSquareTest.php

```diff
@@ -57,7 +57,7 @@ public function providerForPerfectSquare()
             ['0'],
             [81],
             [0],
-            [250],
+            [2500],
```

Comment:
250 is not a perfect square because it cannot be expressed as the product of two equal integers